benchmarks/benchmark-micronaut-data-jdbc/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 micronaut {
-    version '4.0.0-SNAPSHOT'
+    version '4.0.0-M2'
```

Comment:
We should reference the defined version like it’s done for example projects

Reply:
Changed.
However, there are still serde related failures

Reply:
I even tried this https://github.com/micronaut-projects/micronaut-serialization/pull/448 but then hibernate-example-java BookClientSpec.testBookClient fails with different error: 

Client '/books': Error decoding HTTP response body: Error decoding stream for type [interface io.micronaut.data.model.Page]: Unexpected token START_ARRAY, expected START_OBJECT 
 at [Source: (byte[])"[{"id":4,"title":"The Institute","pages":700},{"id":2,"title":"The Shining","pages":600},{"id":1,"title":"The Stand","pages":1000}]"; line: 1, column: 2]
io.micronaut.http.client.exceptions.HttpClientResponseException: Client '/books': Error decoding HTTP response body: Error decoding stream for type [interface io.micronaut.data.model.Page]: Unexpected token START_ARRAY, expected START_OBJECT 
 at [Source: (byte[])"[{"id":4,"title":"The Institute","pages":700},{"id":2,"title":"The Shining","pages":600},{"id":1,"title":"The Stand","pages":1000}]"; line: 1, column: 2]
	at app//io.micronaut.http.client.netty.DefaultHttpClient$FullHttpResponseHandler.makeNormalBodyParseError(DefaultHttpClient.java:2303)
	at

Reply:
a new version of Micronaut Serialization probably needs to be released

Reply:
Current version throws this error

15:31:11.071 [default-nioEventLoopGroup-3-3] ERROR i.m.http.server.RouteExecutor - Unexpected error occurred: No bean introspection available for type [interface io.micronaut.core.convert.value.ConvertibleValues]. Ensure the class is annotated with io.micronaut.core.annotation.Introspected
io.micronaut.core.beans.exceptions.IntrospectionException: No bean introspection available for type [interface io.micronaut.core.convert.value.ConvertibleValues]. Ensure the class is annotated with io.micronaut.core.annotation.Introspected
	at io.micronaut.core.beans.BeanIntrospector.lambda$getIntrospection$3(BeanIntrospector.java:116)
	
while new version will throw the error I pasted above. So it will need additional fixes in serde it seems.